<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>510</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_fullhistory">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fullhistory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>282</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del_history">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_history">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
}

background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>257</y>
      <width>382</width>
      <height>223</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_exp">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>exp</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QPushButton" name="pushButton_ln">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_sin">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_cos">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_tg">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>tg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_sqrt">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_clean_entry">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_clear">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_back">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_div">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_pi">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>π</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_8">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_9">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButton_multi">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_fact">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>n!</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_minus">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_sign">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>±</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QPushButton" name="pushButton_1">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QPushButton" name="pushButton_plus">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_lBrace">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QPushButton" name="pushButton_rBrase">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_0">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButton_point">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="pushButton_equally">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Главное</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Посчитать выражение</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
